internal/sdk/cloudian/sdk.go

```diff
@@ -87,23 +87,22 @@ func fromInternal(g groupInternal) Group {
 	}
 }
 
-type User struct {
-	UserID  string `json:"userId"`
-	GroupID string `json:"groupId"`
-}
+type UserType string
+
+const (
+	UserTypeSystemAdmin UserType = "SystemAdmin"
+	UserTypeGroupAdmin  UserType = "GroupAdmin"
+	UserTypeUser        UserType = "User"
```

Comment:
```suggestion
	UserTypeStd        UserType = "User"
```

?

Reply:
Maybe all of `Standard`, to match lengthy `SystemAdmin` and `GroupAdmin`?

Reply:
Yeah, that works even better. Though longer.